app/src/main/java/org/keepgoeat/presentation/onboarding/OnBoardingAdapter.kt

```diff
@@ -0,0 +1,40 @@
+package org.keepgoeat.presentation.onboarding
+
+import android.content.Context
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import org.keepgoeat.databinding.ItemOnboardingBinding
+
+class OnBoardingAdapter(context: Context) : RecyclerView.Adapter<RecyclerView.ViewHolder>() {
+    private val inflater by lazy { LayoutInflater.from(context) }
+    private var onBoardingList: List<OnBoardingItem> = emptyList()
+
+    class OnBoardingViewHolder(private val binding: ItemOnboardingBinding) :
+        RecyclerView.ViewHolder(binding.root) {
+        fun bind(onboarding: OnBoardingItem) {
+            binding.tvOnboardingTitle.setText(onboarding.title)
+            binding.tvOnboardingDes.setText(onboarding.des)
+            binding.ivOnboarding.setImageDrawable(binding.root.context.getDrawable(onboarding.image))
+        }
+    }
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): RecyclerView.ViewHolder {
```

Comment:
멀티 뷰홀더를 사용할 것이 아니라면 또는 사용할 예정이 없다면 OnBoardingViewHolder 로 바꾸는 것이 좋을 것 같아용~
```suggestion
    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): OnBoardingViewHolder {
```